packages/svelte/src/internal/client/runtime.js

```diff
@@ -815,7 +841,7 @@ export function get(signal) {
 		) {
 			if (current_dependencies === null) {
 				current_dependencies = [signal];
-			} else {
+			} else if (current_dependencies[current_dependencies.length - 1] !== signal) {
```

Comment:
What's this change about?

Reply:
Avoids adding the same signal a thousand times. Nice perf win

Reply:
ok so unrelated thing you came across but was good to fix 👍